lib/screens/exhibition_screen.dart

```diff
@@ -1,25 +1,67 @@
 import "package:flutter/material.dart";
 import "package:flutter_widget_from_html_core/flutter_widget_from_html_core.dart";
+import "package:noheva_api/noheva_api.dart";
+import "package:noheva_visitor_ui/database/dao/layout_dao.dart";
+import "package:noheva_visitor_ui/database/dao/page_dao.dart";
+import "package:noheva_visitor_ui/main.dart";
+import "package:noheva_visitor_ui/screens/default_screen.dart";
 import "package:noheva_visitor_ui/utils/html_widgets.dart";
+import "package:simple_logger/simple_logger.dart";
 
 /// Exhibition Screen
 ///
-/// Displays Exhibition content
+/// Displays Exhibition [exhibitionId] content
 class ExhibitionScreen extends StatefulWidget {
-  const ExhibitionScreen({Key? key}) : super(key: key);
+  final String exhibitionId;
+
+  const ExhibitionScreen({Key? key, required this.exhibitionId})
+      : super(key: key);
 
   @override
   State<ExhibitionScreen> createState() => _ExhibitionScreenState();
 }
 
 /// Exhibition screen state
 class _ExhibitionScreenState extends State<ExhibitionScreen> {
```

Comment:
Rename ExhibitionScreen to PageScreen, this does not display an exhibition, it displaying single page in an Exhibition

Reply:
Resolved.